app/Http/Controllers/User/MyRequestController.php

```diff
@@ -81,6 +84,19 @@ public function update(Request $request, $id)
             $this->notification->store($data);
         }
 
+        if ($request->status == 'returning') {
+            $bookId = $this->bookUser->findId($id)->book_id;
+            $countReturnedBook = $this->bookUser->countReturned($bookId);
+            $findReturned = $this->bookMeta->findReturned($bookId);
+            if (empty($findReturned->key)) {
+                $this->bookMeta->insertReturned([
+                    'key' => 'count_returned',
+                    'value' => $countReturnedBook->book_returned,
+                    'book_id' =>$bookId,
+                ]);
+            } else $this->bookMeta->updateReturned($bookId, ['value' => $countReturnedBook->book_returned]);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.ControlStructures.InlineControlStructure.NotAllowed</br>**Severity**: error</br>**File**: app/Http/Controllers/User/MyRequestController.php</br>**Line**: 97</br>Inline control structures are not allowed